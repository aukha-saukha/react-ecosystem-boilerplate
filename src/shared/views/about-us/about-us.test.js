import { shallow } from 'enzyme';
import React from 'react';

import AboutUs from './about-us.react';

describe('AboutUs component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<AboutUs />);

    expect(wrapper).toMatchSnapshot();
  });
});
