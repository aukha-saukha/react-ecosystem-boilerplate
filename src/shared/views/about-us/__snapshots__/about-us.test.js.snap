// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component Renders correctly 1`] = `
<ContextProvider
  value={
    Array [
      "en-us",
      [Function],
    ]
  }
>
  <AboutUs />
</ContextProvider>
`;
